topics/modules-api-ref.md

```diff
@@ -1053,6 +1053,8 @@ The input flags are:
 
     REDISMODULE_ZADD_XX: Element must already exist. Do nothing otherwise.
     REDISMODULE_ZADD_NX: Element must not exist. Do nothing otherwise.
+    REDISMODULE_ZADD_LT: For existing element, only update if the new score is less than the current score.
+    REDISMODULE_ZADD_GT: For existing element, only update if the new score is greater than the current score.
```

Comment:
this change isn't needed, auto generated when we make a release.
but i suppose we can merge it, it'll get overwritten later.

Reply:
Interesting.  If it's auto-generated, then it seems to be missing some flags.

Reply:
we (manually) auto generate it before publishing a new release. so maybe what's missing are flags that were not released yet?
which flags are missing?